python/ray/tune/tests/test_progress_reporter.py

```diff
@@ -246,21 +246,24 @@ def testProgressStr(self):
             t.trial_id = "%05d" % i
             t.local_dir = "/foo"
             t.location = "here"
-            t.config = {"a": i, "b": i * 2}
-            t.evaluated_params = t.config
+            t.config = {"a": i, "b": i * 2, "n": {"k": [i, 2 * i]}}
+            t.evaluated_params = ["a", "b", "n/k/0", "n/k/1"]
             t.last_result = {
                 "config": {
                     "a": i,
-                    "b": i * 2
+                    "b": i * 2,
+                    "n": {
+                        "k": [i, 2 * i]
+                    }
```

Comment:
I didn't even realize we supported lists.

Reply:
They come up in examples such as #9307. I also tested this with a manual config/constant values, and it still works. The flattened list lookup comes from `flatten_resolved_vars()`.
I amended the PR to allow lookup of other Sequences, too, and fixed a small bug where a lookup of `a/b/c` would return `lookup["a"]["b"]` if `lookup["a"]["b"]["c"]` did not exist. In this case, `default` should be returned.